cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,120 @@
 /// <reference types='cypress' />
 
+const { generateWorker } = require('../support/workerGenerate');
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  const { firstName, lastName, email, age, salary, department } = generateWorker();
+
+  it('should contain pagination on the page', () => {
+    cy.get('.pagination-bottom').should('exist');
+    cy.get('.pagination-bottom').should('contain', 'Previous');
+    cy.get('.pagination-bottom').should('contain', 'Page');
+    cy.get('.pagination-bottom').should('contain', 'of');
+    cy.get('.pagination-bottom').should('contain', 'rows');
+    cy.get('.pagination-bottom').should('contain', 'Next');
+  });
+
+  it('should be able to select the count of rows', () => {
```

Comment:
assert not only the numbers in the dropdown but also the table rows. For example you change row quantity then asserting that table contain selected quantity

Reply:
Fixed

Reply:
you still checking the number of selected rows in the dropdown. Flow should be the next:
1. Select 5 rows fro the dropdown
2. Assert that table has 5 rows(now you assert that dropdown has "5 rows" value etc)

Reply:
Fixed.